src/components/Login/Web3Login.tsx

```diff
@@ -167,7 +178,7 @@ const Web3Login: FC<Props> = ({
 			const injectedWindow = window as Window & InjectedWindow;
 
 			const wallet = isWeb3Injected
-				? injectedWindow.injectedWeb3[chosenWallet]
+				? injectedWindow.injectedWeb3[chosenWallet === Wallet.POLKASAFE ? Wallet.POLKADOT : chosenWallet]
```

Comment:
this will only allow the users to use polkadot-js wallet to login with Polkasafe. Please let users use other wallets too. Polkasafe will allow other wallets to login with in the future.